README.md

```diff
@@ -24,24 +24,17 @@ Projects listed on this page are open source that Microsoft maintain or contribu
+|  | [OPA Gatekeeper](https://github.com/open-policy-agent/gatekeeper) <br> ([releases](https://github.com/open-policy-agent/gatekeeper/releases)) | K8s native Open Policy Agent policy enforcement | CNCF: graduated <br> API: Config: v1alpha1; ConstraintTemplate: v1beta1; Constraints: v1beta1 | [#kubernetes-policy](https://openpolicyagent.slack.com/archives/CDTN970AX) <br> [GitHub issues](https://github.com/open-policy-agent/gatekeeper/issues) | <ul><li>[Azure Policy for AKS](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/rego-for-aks)</ul><li>[Azure Policy for AKS Engine](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/aks-engine)</li><li>[Azure Policy for Azure Arc connected clusters](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/policy-for-kubernetes#install-azure-policy-add-on-for-azure-arc-enabled-kubernetes)</li><ul> |
+|  | [Secrets Store CSI Driver](http://sigs.k8s.io/secrets-store-csi-driver) <br> ([releases](https://github.com/kubernetes-sigs/secrets-store-csi-driver/releases)) <br> [Builds](https://testgrid.k8s.io/sig-auth-secrets-store-csi-driver#secrets-store-csi-driver-e2e-vault-postsubmit) | Integrates secrets stores with Kubernetes via a [Container Storage Interface (CSI)](https://kubernetes-csi.github.io/docs/) volume | Kubernetes: beta <br> API: v1alpha1 | [#csi-secrets-store](https://kubernetes.slack.com/messages/csi-secrets-store) <br> [GitHub issues](https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues) |  |
+|  | [Azure KeyVault Provider for Secrets Store CSI Driver](https://github.com/Azure/secrets-store-csi-driver-provider-azure) <br> ([releases](https://github.com/Azure/secrets-store-csi-driver-provider-azure/releases)) | Enables mounting AKV secrets as volumes in K8s pods | Azure: incubation <br> API: N/A | [GitHub issues](https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues) | [Use with AKS](https://docs.microsoft.com/en-us/azure/aks/developer-best-practices-pod-security#use-azure-key-vault-with-secrets-store-csi-driver) |
 | Cloud Native Service Mesh |  |  |  |  |
-|  | [Service Mesh Interface (SMI) Spec](https://smi-spec.io/) | A standard interface for service meshes on Kubernetes | CNCF: sandbox <br> APIs: Traffic Split v1alpha3; Traffic Target v1alpha1; Traffic Spec v1alpha2; Traffic Metrics v1alpha1 | [#smi](https://cloud-native.slack.com/messages/smi) <br> [GitHub issues](https://github.com/deislabs/smi-spec/issues) |  |
+|  | [Service Mesh Interface (SMI) Spec](https://smi-spec.io/) | A standard interface for service meshes on Kubernetes | CNCF: sandbox <br> APIs: Traffic Access Control: v1alpha3; Traffic Metrics: v1alpha1; Traffic Specs: v1alpha4; Traffic Split: v1alpha4  | [#smi](https://cloud-native.slack.com/messages/smi) <br> [GitHub issues](https://github.com/deislabs/smi-spec/issues) |  |
```

Comment:
can we add bullet points to APIs?

Reply:
@sozercan I tried bullet points before. it will make the height of that row really big such that the table looks terrible.